docs/guides/getting-started-orchestrate-apis.md

```diff
@@ -10,11 +10,18 @@ keywords:
 <span class="badge badge--beginner">Beginner</span>
 <span class="badge badge--medium">Time estimate: 15 minutes</span>
 
-A Connector is a reusable building block that works out of the box. Each Connector task can be configured with domain-specific parameters without implementing custom business logic.
+This guide will walk you through working with a REST Connector task as a first time Camunda 8 user.
 
-The concept of a Connector consists of two parts: the business logic is implemented as a job worker, and the user interface during modeling is provided using an element template.
+:::note
+New to Connectors? Review our [introduction to Connectors](/docs/components/connectors/introduction.md) to get familiar with their capabilities.
+:::
 
-This guide will walk you through working with a REST Connector task as a first time Camunda 8 user.
+A **Connector** is a reusable building block that performs integration with an external system to orchestrate and complete work. For example, you can notify of pending or completed processes through communication tools such as [Slack](/docs/components/connectors/out-of-the-box-connectors/slack.md) or [Microsoft Teams](/docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md), and create or update records in enterprise business systems.
+
+A Connector consists of two parts:
+
+1. The programming code in Java to connect to the external system.
+2. The user interface to be used during modeling, which is provided using [Connector templates](/docs/components/connectors/manage-connector-templates.md).
```

Comment:
This should be more specific to the REST Connector, but may require some assistance from the Connectors team. Having a description of API orchestration would be good as well.

Here are the questions I'm looking to have answered in the intro:

* What is API Orchestration?
* How does it relate to the REST Connector?
* What skills do I need to be successful with this Guide?

Reply:
Let me know if this is clear based upon my latest commit 👍